docs/source/user_guide/12_list_of_environment_variables.rst

```diff
@@ -0,0 +1,48 @@
+List of Env vars
+_________________
+
+.. list-table:: User List
+   :widths: 35 100 20 20
+   :header-rows: 1
+
+   * - Environment Variable Name
+     - Description
+     - Default value
+     - Is_deprecated
+     - Importance
+   * - .. envvar:: RADICAL_UTILS_NO_ATFORK
```

Comment:
The purpose of a directive like `.. envvar::` is to define a role that you will use later. It should appear on a line by itself exactly once in the project. Depending on what makes the most sense, you could have a list of directives to define environment variables in a single rst document, or, if there is a clear "owner" of the env var, you could put the directive in the corresponding module documentation.

Then, if it is helpful, you could create tables like these that use the roles (not the directives) to identify the env vars.

In such an approach, the repeated references could make more sense:
* at the `.. envvar:: RADICAL_UTILS_NO_ATFORK` you would provide a description of what the environment variable means and does.
* In a pair of tables that describe environment variables relevant to users or developers, you would focus your descriptive text on the target audience, and why they would want to use the particular env var.

In any case, it probably makes sense to first focus on getting the `.. envvar::` directives into the package _somewhere_ so that the documentation exists and is referentiable. _Then_, if tables like these are necessary, they can be tackled afterwards.

Reply:
> does not have 12_list_of_environment_variables in the toc list
Added now in toctree. 

>In any case, it probably makes sense to first focus on getting the .. envvar:: directives into the package somewhere so that the documentation exists and is referentiable
I am trying to create a page where we'll have this .. envvar:: directives and then we can refer to those variable throughout the codebase. I do understand the definitions .. envvar:: directives should be unique in one place, else sphinx picks the last definition it finds and link to that definition. 
I sure there is more to this, you are saying and I ain't getting all of it.